jboss/container/eap/launch/added/openshift-launch.sh

```diff
@@ -3,43 +3,9 @@
 # Always start sourcing the launch script supplied by wildfly-cekit-modules
 source ${JBOSS_HOME}/bin/launch/launch.sh
 
-management_port=""
-if [ -n "${PORT_OFFSET}" ]; then
-  management_port=$((9990 + PORT_OFFSET))
-fi
-
-# TERM signal handler
-function clean_shutdown() {
-  log_error "*** JBossAS wrapper process ($$) received TERM signal ***"
-  if [ -z ${management_port} ]; then
-    $JBOSS_HOME/bin/jboss-cli.sh -c "shutdown --timeout=60"
-  else
-    $JBOSS_HOME/bin/jboss-cli.sh --commands="connect remote+http://localhost:${management_port},shutdown --timeout=60"
-  fi
-  wait $!
-}
-
 function runServer() {
   local instanceDir=$1
-
-  # exposed by wildfly-cekit-modules
-  configure_server
-
-  log_info "Running $JBOSS_IMAGE_NAME image, version $JBOSS_IMAGE_VERSION"
-
-  trap "clean_shutdown" TERM
```

Comment:
Do we still trap these somewhere else? What does ctrl-c do now when the server is running?

Reply:
That has been moved to wildfly-cekit-modules already: https://github.com/wildfly/wildfly-cekit-modules/blob/master/jboss/container/wildfly/launch-config/config/added/launch/launch.sh#L103

Reply:
Ah, OK, thanks!